cmd/dep/godep_importer_test.go

```diff
@@ -395,3 +338,13 @@ func equalImports(a, b []godepPackage) bool {
 
 	return true
 }
+
+// projectRootPtr takes a ProjectRoot value and returns a pointer.
+func projectRootPtr(p gps.ProjectRoot) *gps.ProjectRoot {
```

Comment:
Alright! I'll adjust the tests according to your comments @carolynvs. Will to that within a week, I'm currently on vacation. Thanks for the feedback!